cmd/otelcol/config/collector/fargate_config.yaml

```diff
@@ -0,0 +1,154 @@
+# This collector config file is designed for use within an ECS task.
+# The collector should run in a sidecar container within an ECS task.
+config_sources:
+  env:
+    defaults:
+      METRICS_TO_EXCLUDE: []
+      IMAGES_TO_EXCLUDE: ["quay.io/signalfx/splunk-otel-collector"]
+
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+  http_forwarder:
+    ingress:
+      endpoint: 0.0.0.0:6060
+    egress:
+      endpoint: "https://api.${SPLUNK_REALM}.signalfx.com"
+  zpages:
+    endpoint: 0.0.0.0:55679
+  smartagent:
+
+receivers:
+  jaeger:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:14250
+      thrift_binary:
+        endpoint: 0.0.0.0:6832
+      thrift_compact:
+        endpoint: 0.0.0.0:6831
```

Comment:
Are these needed for Fargate?

Reply:
Maybe sidecar requires this -- if so readme should call out this should be removed in other deployment model

Reply:
No, I removed them.